modules/reactor/test/tests/meshgenerators/patterned_cartesian_mesh_generator/tests

```diff
@@ -180,14 +180,6 @@
       expect_err = 'All the meshes provided in inputs must be used here.'
       requirement ='The system shall throw an error if not all the cartesian meshes provided in inputs are used in pattern.'
     []
-    [err_no_appropriate_pitch_meta]
-      type = 'RunException'
-      input = 'patterned_pattern.i'
-      cli_args = 'Mesh/pattern_2/generate_core_metadata=false
-                 --mesh-only "patterned_err.e"'
-      expect_err = 'the unit square input mesh does not contain appropriate meta data required for generating an assembly.'
```

Comment:
we cant delete requirements pretty much. @cticenhour for backing this up

let's modify this test to still make it work

Reply:
I would agree with this - requirements can be tweaked, expanded, refined, etc but the core idea of the requirement should not be removed unless it is truly obsolete. In this case, if the error triggered by the test can no longer be hit by the user, that could be justification, but is that what is happening here?